src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.storage.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    public static final String FIRST_LINE = "fruit,quantity";
+
+    @Override
+    public String getReport() {
+        StringBuilder resultingReport = new StringBuilder();
+        resultingReport.append(FIRST_LINE);
+        for (Fruit fruit : Storage.fruits) {
```

Comment:
Accessing the `fruits` field in `Storage` directly violates the encapsulation principle. It would be better to provide a method in the `Storage` class to get the necessary data.